src/components/templates/PosterHall/PosterHall.tsx

```diff
@@ -46,6 +46,7 @@ export const PosterHall: React.FC<PosterHallProps> = ({ venue }) => {
         searchInputValue={searchInputValue}
         liveFilterValue={liveFilter}
         setLiveValue={setLiveFilter}
+        searchTitleBase={venue?.chatTitle}
```

Comment:
`venue.chatTitle` is being used for the side chat. Check `/hooks/chatSidebar.ts`. I don't thing the placeholder of posterhall and the sidechat title should use the same value.

Reply:
Agreed